Makefile.am

```diff
@@ -25,14 +25,15 @@ EXTRA_DIST = \
 	ovirt-web-ui.spec.in \
 	README.md \
 	LICENSE \
+	branding \
 	src \
 	config \
 	doc \
 	scripts \
 	packaging \
 	package.json \
 	index.html \
-  index.jsp \
+    index.jsp \
```

Comment:
Absolutely. And do you disagree with it in this pull or in general? I mean it looks like an indentation fix to me.